tests/EdgeToEdge/Routes/ApplyForMembershipRouteTest.php

```diff
@@ -620,4 +528,8 @@ private function insertIncentives( FunFunFactory $factory, Incentive ...$incenti
 		}
 		$em->flush();
 	}
+
+	private function insertPaymentId( FunFunFactory $factory, int $index ): void {
```

Comment:
Manually inserting a payment ID should no longer be necessary, it's part of the test environment setup in `tests/RebuildDatabaseSchemaTrait.php`

Reply:
Ah great, I was waiting for a standup to ask about the best way to do this with the test setup.